docs/backend/upgrading/version-specific-migration/upgrade-to-61.md

```diff
@@ -117,3 +117,44 @@ If you have an existing Plone 5.2 or 6.0 site and you migrate to 6.1, then migra
 -   If the `plone.app.discussion` Python package is _not_ in your setup, but the site has existing comments (discussions), then the migration code stops with an error.
     Apparently you _were_ using comments in your site.
     Add the `plone.app.discussion` package to your dependencies, and run the migration again.
+
+
+## Distributions
+
+Plone 6.1 introduces the concept of a Plone {term}`distribution`.
+A Plone distribution is a Python package that defines specific features, themes, add-ons, and configurations that get activated when creating a Plone site.
+Now it is available in core Plone as the recommended way for {doc}`creating a new Plone site </install/add-site>`.
```

Comment:
I found no way to actually run `plone.distribution` at the suggested page, as the target states:

```
assumes that you have already {doc}`installed </install/index>` and started Plone.
```

...and none of those options pull in a 6.1 site, and neither in the currently open PRs for docs.

What did I miss?

Reply:
@stevepiercy You don't "run plone.distribution" per se. It's just there if you install the `Plone` package (which is what all the documented install methods do) rather than the more minimal `Products.CMFPlone` package.

It's by design that the install methods as documented still use Plone 6.0, since Plone 6.1 is still pre-final release.

With the Cookieplone project template, be sure to choose Plone 6.1.0b1:
```
  [7/17] Should we use prerelease versions? (No): Yes
  [8/17] Plone Version (6.1.0b1): 
```

With buildout- or pip-based installs, be sure to use `https://dist.plone.org/release/6.1-latest/` instead of `https://dist.plone.org/release/6-latest/`

Reply:
I've updated the checklist for updates to docs for Plone 6.1 final:

https://github.com/plone/documentation/issues/1721